src/lib/visual-generator.ts

```
@@ -0,0 +1,367 @@
+          height: 2 + Math.random() * 4,
+          color,
+          opacity: 0.8 + Math.random() * 0.2
+        };
+      case 'text':
+        const texts = ['AI', '🧠', '💭', '✨', '🌟', '💡', '🔮', '🎯'];
+        return {
+          type: 'text',
+          x: Math.random() * width,
+          y: Math.random() * height,
+          color,
+          text: texts[Math.floor(Math.random() * texts.length)],
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)